src/compiler/sys.ts

```diff
@@ -869,6 +875,63 @@ namespace ts {
             };
             return nodeSystem;
 
+            /**
+             * Uses the builtin inspector APIs to capture a CPU profile
+             * See https://nodejs.org/api/inspector.html#inspector_example_usage for details
+             */
+            function enableCPUProfiler(path: string, cb: () => void) {
+                if (activeSession) {
+                    cb();
+                    return false;
+                }
+                const inspector: typeof import("inspector") = require("inspector");

```

Comment:
@weswigham don't forget to add `inspector: false` to `package.json`. these node apis otherwise break bundling resolution. I'm kinda waiting for `3.7` to be webpack-compatible without extra config. this `require` call will break it again.